README.md

```diff
@@ -1 +1,15 @@
-# .github
\ No newline at end of file
+<p align="center" >
+<img src="https://docs.dash.org/en/stable/_images/dash_logo.png" alt="Dash Wallet logo" title="Dash Wallet" width="300">
+</p>
+
+WELCOME TO DASHPAY!!
+
+The DashPay organization houses all GitHub activity related to Dash.
+
+Previously, all development related to Dash Platform happened under the dashevo GitHub. However, we have since archived the repositories in Dashevo and moved everything to the DashPay GitHub.
+
+By consolidating all Dash-related activities under one organization, we aim to simplify the process of developer contribution to both Dash Core and Dash Platform.
+
+[Vist our website](https://www.dash.org/) to know more about Dash. 
+
+See the [Dash Platform documentation](https://dashplatform.readme.io/docs) for further information about Dash's Web3 platform.
```

Comment:
Let's include a link to the core documentation as well since this organization is for everything